src/Controls/tests/Xaml.UnitTests/Issues/Maui32597.xaml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:local="clr-namespace:Microsoft.Maui.Controls.Xaml.UnitTests"
+             x:Class="Microsoft.Maui.Controls.Xaml.UnitTests.Maui32597">
```

Comment:
We are missing `x:DataType="local:Maui32597ViewModel"` so these bindings won't be compiled by XamlC and SourceGen